.github/ISSUE_TEMPLATE/feature-request.md

```diff
@@ -0,0 +1,31 @@
+---
+name: "\U0001F680 Feature Request"
+about: Suggest an idea for this project
+labels: feature-request, needs-triage
+---
+
+<!--- Provide a general summary of the issue in the Title above -->
+
+## Describe the Feature
+<!--- A clear and concise description of the feature you are proposing -->
+
+### Is your Feature Request related to a problem?
+<!--- A description of the issue, e.g. I'm always frustrated when... -->
+
+## Proposed Solution
+<!--- Not required, but suggest how to implement the addition or change -->
+
+### Describe alternatives you've considered
+<!--- Any alternative solutions or features you've considered -->
+
+## Additional Context
+<!--- How has this issue affected you? What are you trying to accomplish? -->
+<!--- Providing context helps us come up with a solution that is most useful in the real world -->
+
+
+* [ ] :wave: I may be able to implement this feature request
+* [ ] :warning: This feature might incur a breaking change
```

Comment:
If they check this box unless it's a ridiculously critical bug, we are not going to take it. I would warn them to not try to supply a prototype/start implementation until they get some initial feedback from the team.